exchanges-connector/src/main/kotlin/fund/cyber/markets/connector/ConnectorRunner.kt

```diff
@@ -20,14 +26,19 @@ import info.bitrich.xchangestream.hitbtc.HitbtcStreamingExchange
 import info.bitrich.xchangestream.okcoin.OkCoinStreamingExchange
 import info.bitrich.xchangestream.okcoin.OkExStreamingExchange
 import info.bitrich.xchangestream.poloniex2.PoloniexStreamingExchange
+import org.slf4j.LoggerFactory
 import org.springframework.beans.factory.BeanFactory
 import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.context.ApplicationContext
 import org.springframework.retry.support.RetryTemplate
 import org.springframework.stereotype.Component
 
+val XCHANGE_TRADE_CONNECTOR_CLASS = XchangeTradeConnector::class.java
+val XCHANGE_ORDERBOOK_CONNECTOR_CLASS = XchangeOrderbookConnector::class.java
+
 @Component
 class ConnectorRunner {
+    val log = LoggerFactory.getLogger(javaClass)!!
 
     @Autowired
     private lateinit var beanFactory: BeanFactory
```

Comment:
You don't need autowire beanFactory. You could get it from applicationContext

Reply:
done